e2e/resources/workbench-config.ts

```diff
@@ -24,59 +25,59 @@ const urlPath = {
 
 // localhost development server
 const local = {
-  uiBaseUrl: process.env.DEV_LOGIN_URL || 'http://localhost:4200',
-  apiBaseUrl: process.env.DEV_API_URL || 'http://localhost/v1',
-  userEmailDomain: '@fake-research-aou.org',
-  collaboratorUsername: process.env.COLLABORATOR_USER || 'puppetmaster@fake-research-aou.org',
-  writerUserName: process.env.WRITER_USER || 'puppetmaster@fake-research-aou.org',
-  readerUserName: process.env.READER_USER || 'puppetcitester1@fake-research-aou.org',
-  defaultCdrVersionName: 'Synthetic Dataset v3',
```

Comment:
Please restore the word "name" to the cdr version variables.  It's important to know whether it's a name or ID.

Reply:
"version" by itself does not give us enough context.

Reply:
Done.